src/components/events/eventsMain.tsx

```diff
@@ -34,8 +34,12 @@ const EventsMain = ({ userRole, setEventToDisplay }: eventProps) => {
 
   const handleDeleteEvent = async (event: IEvent) => {
     const { status } = await deleteEvent(event._id);
-    if (status !== StatusCodes.OK) console.log('could not delete event');
-    else fetchEvents();
+    if (status !== StatusCodes.OK) {
+      handleAlert(false, 'לא הצלחנו למחוק את האירוע');
+    } else {
+      handleAlert(true, 'האירוע נמחק בהצלחה');
+      fetchEvents();
+    }
   };
 
   const sortByDate = (events: IEvent[]) => events
```

Comment:
```
   .sort((a: IEvent, b: IEvent) =>  
a.dates[0] < b.dates[0] || a.dates[1] < b.dates[1] ? : a.dates[0] > b.dates[0] || a.dates[1] > b.dates[1]?-1:0);
```

or at least  you know... if else

Reply:
is else is not necassary because I return in every if.
My linter does not like your suggestion because it is a nested trinary condition (I started with it but my current code is more readable in my opinion).